concepticondata/conceptlists/Oskolskaya-2021-254.tsv

```diff
@@ -0,0 +1,255 @@
+Oskolskaya-2021-254-43	43	black	163	BLACK
+Oskolskaya-2021-254-44	44	navel (n.)	1838	NAVEL
+Oskolskaya-2021-254-45	45	stand (v.)	1442	STAND
+Oskolskaya-2021-254-46	46	bite (v.)	1403	BITE
+Oskolskaya-2021-254-47	47	back (n.)	1291	BACK
+Oskolskaya-2021-254-48	48	wind (n.)	960	WIND
+Oskolskaya-2021-254-49	49	smoke (n.)	778	SMOKE (EXHAUST)
+Oskolskaya-2021-254-50	50	what?	1236	WHAT
+Oskolskaya-2021-254-51	51	child (kin term) (n.)	1801	CHILD (DESCENDANT)
+Oskolskaya-2021-254-52	52	egg (n.)	744	EGG
+Oskolskaya-2021-254-53	53	give (v.)	1447	GIVE
+Oskolskaya-2021-254-54	54	new	1231	NEW
+Oskolskaya-2021-254-55	55	burn (intr.) (v.)	2102	BURN
+Oskolskaya-2021-254-56	56	not	1240	NOT
+Oskolskaya-2021-254-57	57	good	1035	GOOD
+Oskolskaya-2021-254-58	58	know (v.)	1410	KNOW (SOMETHING)
+Oskolskaya-2021-254-59	59	knee (n.)	1371	KNEE
+Oskolskaya-2021-254-60	60	sand (n.)	671	SAND
+Oskolskaya-2021-254-61	61	laugh (v.)	1355	LAUGH
+Oskolskaya-2021-254-62	62	hear (v.)	1408	HEAR
+Oskolskaya-2021-254-63	63	soil (earth) (n.)	1228	EARTH (SOIL)
+Oskolskaya-2021-254-64	64	leaf (n.)	628	LEAF
+Oskolskaya-2021-254-65	65	red	156	RED
+Oskolskaya-2021-254-66	66	liver (n.)	1224	LIVER
+Oskolskaya-2021-254-67	67	hide (v.)	602	HIDE (CONCEAL)
+Oskolskaya-2021-254-68	68	skin (hide) (n.)	763	SKIN
+Oskolskaya-2021-254-69	69	suck (v.)	1421	SUCK
+Oskolskaya-2021-254-70	70	carry (v.)	700	CARRY
+Oskolskaya-2021-254-71	71	ant (n.)	587	ANT
+Oskolskaya-2021-254-72	72	heavy	1210	HEAVY
+Oskolskaya-2021-254-73	73	take (v.)	1749	TAKE
+Oskolskaya-2021-254-74	74	old	2113	OLD (USED)
+Oskolskaya-2021-254-75	75	eat (v.)	1336	EAT
+Oskolskaya-2021-254-76	76	thigh (n.)	471	UPPER LEG (THIGH)
+Oskolskaya-2021-254-77	77	thick	1244	THICK
+Oskolskaya-2021-254-78	78	long	1203	LONG
+Oskolskaya-2021-254-79	79	blow (v.)	175	BLOW (OF WIND)
+Oskolskaya-2021-254-80	80	wood (n.)	1803	WOOD
+Oskolskaya-2021-254-81	81	run (v.)	1519	RUN
+Oskolskaya-2021-254-82	82	fall (v.)	1280	FALL
+Oskolskaya-2021-254-83	83	eye (n.)	1248	EYE
+Oskolskaya-2021-254-84	84	ash (n.)	646	ASH
+Oskolskaya-2021-254-85	85	tail (n.)	1220	TAIL
+Oskolskaya-2021-254-86	86	dog (n.)	2009	DOG
+Oskolskaya-2021-254-87	87	cry / weep (v.)	1839	CRY
+Oskolskaya-2021-254-88	88	tie (v.)	1917	TIE
+Oskolskaya-2021-254-89	89	see (v.)	1409	SEE
+Oskolskaya-2021-254-90	90	sweet	717	SWEET
+Oskolskaya-2021-254-91	91	rope (n.)	1218	ROPE
+Oskolskaya-2021-254-92	92	shade / shadow (n.)	1388	SHADE
+Oskolskaya-2021-254-93	93	bird (n.)	937	BIRD
+Oskolskaya-2021-254-94	94	salt (n.)	1274	SALT
+Oskolskaya-2021-254-95	95	small	1246	SMALL
+Oskolskaya-2021-254-96	96	wide	1243	WIDE
+Oskolskaya-2021-254-97	97	star (n.)	1430	STAR
+Oskolskaya-2021-254-98	98	in	1460	IN
+Oskolskaya-2021-254-99	99	hard	1884	HARD
+Oskolskaya-2021-254-100	100	crush / grind (v.)	1033	GRIND
+Oskolskaya-2021-254-101	101	mountain (hill) (n.)	2118	MOUNTAIN OR HILL
+Oskolskaya-2021-254-102	102	sit (v.)	1416	SIT
+Oskolskaya-2021-254-103	103	(finger)nail (n.)	1258	FINGERNAIL
```

Comment:
Not sure about this one. The source also has _claw_ here. But `2128 CLAW OR NAIL` also doesn't seem to fit.

Reply:
The rule would be: what we find in Savelyev's concept list.